AutomatedTesting/Gem/PythonTests/WhiteBox/tests/WhiteBox_AddComponentToEntity.py

```diff
@@ -18,21 +18,19 @@ class Tests():
 
 
 def C28798177_WhiteBox_AddComponentToEntity():
-    import os
-    import sys
-    from Gems.WhiteBox.Editor.Scripts import WhiteBoxInit as init
 
+    import editor_python_test_tools.hydra_editor_utils as hydra
     import azlmbr.bus as bus
     import azlmbr.editor as editor
     import azlmbr.legacy.general as general
 
+    from Gems.WhiteBox.Editor.Scripts import WhiteBoxInit as init
     from editor_python_test_tools.utils import Report
-    
     from editor_python_test_tools.utils import TestHelper as helper
 
     # open level
     helper.init_idle()
-    helper.open_level("WhiteBox", "EmptyLevel", no_prompt=False)
+    hydra.open_base_level()
```

Comment:
this opens a different level. is that the intention of this change? it also does the init_idle()
def open_base_level():
    helper.init_idle()
    helper.open_level("", "Base")

Reply:
Yeah the idea is that we want to move away from having tests rely on pre-constructed levels, since we can re-create what we need programmatically in the test itself. In this case (as the name hints at), the `WhiteBox/EmptyLevel` is just a default level setup.

You are correct though, the `helper.init_idle()` lines above can be removed since that is done in `open_base_level`

Also @Broganab I just did a quick search and it looks like these are the last tests that use the `WhiteBox/EmptyLevel` so I think you can go ahead and remove it as well with this change! :)

Reply:
Good catch on the init_idle() dupe.